src/main/app/form/validation.ts

```diff
@@ -89,6 +89,19 @@ export const isInvalidHelpWithFeesRef: Validator = value => {
   }
 };
 
+export const isInvalidPostcode: Validator = value => {
+  const fieldNotFilledIn = isFieldFilledIn(value);
+  if (fieldNotFilledIn) {
+    return fieldNotFilledIn;
+  }
+
+  if (typeof value === 'string') {
```

Comment:
this value will definitely be a string if it is filled in so you don't need this check

Reply:
👍 Good point, I've cast this to a string